import { generateError } from 'src/errors/generate-error';

export const WalletAlreadyConnectedError = generateError();
